frontend/front/src/pages/Public/Components/Button/ButtonGetPump.js

```diff
@@ -15,21 +18,23 @@ const ButtonGetPumnp = ({ variant }) => {
         variant="Button"
         onClick={() => window.scrollTo(0, 0)}
         sx={{
+          px: 2,
           color: "var(--color-text-2)",
           background: "var(--bgColor-1)",
-          fontSize: "1rem",
           height: "50px",
           fontWeight: 600,
           borderRadius: "50px",
+          width: "auto",
           minWidth: 200,
           transition: "0.3s cubic-bezier(.47,1.64,.41,.8)",
           letterSpacing: "-.03em",
+          textAlign: "center",
           "&:hover": {
             background: "var(--bgColor-1)",
           },
         }}
       >
-        GET A HEAT PUMP
+        {t("public.global-labels.button-get-heat-pump")}
```

Comment:
I'm not totally familiar with how this works, but would it be possible to store these as constants somewhere? It seems like accidentally adding a typo to one of these keys would be super bad.

Reply:
Yes, all the texts are stored in the locales folder. In that folder, there are four JSON files, one for each language, containing the texts. All the text is stored under the translation namespace, which is located in the i18n.js file within the resources object. 
Doc ref: https://react.i18next.com/guides/quick-start

Reply:
@mattdelsordo I agree that constants would be helpful but it could be tough to maintain? If we changed the name of a key, the constant would no longer point to the right place and that could be a hard-to-track-down bug.

Not sure if there is maybe a way to programmatically generate constants, or a map of constants, from the .json files in the locales folder, that would update automatically if the keys were changed?

Reply:
Fixed it: https://github.com/codeforboston/urban-league-heat-pump-accelerator/pull/489/commits/2a089cd364d1b502c4ae3c522afa5393be429f5e